sign-pkg-mac/action.yaml

```diff
@@ -111,8 +105,10 @@ runs:
         done
 
         # don't forget the application itself
-        cp -a ".app/$app_name.app" "$volpath/"
-        echo "app_path=$volpath/$app_name.app" >> "$GITHUB_ENV"
+        artifact_app="$(ls -dt .app/*.app | head -n 1)"
+        cp -a "$artifact_app" "$volpath/"
+        app_name="$(basename "$artifact_app")"
+        echo "app_path=$volpath/$app_name" >> "$GITHUB_ENV"
```

Comment:
seems like this should use `"app_path=$volpath/$artifact_app"` if I'm understanding correctly here

Reply:
No - artifact_app is the relative pathname including the .app/ subdirectory path. Line 110 takes the basename of artifact_app, and that's where we'll find the target of line 109's copy operation.

Reply:
sorry, my mistake.  I mixed up precisely what basename was doing.  off to get more ☕